app/src/main/java/com/example/paritosh/stormy/WeatherActivity.java

```diff
@@ -10,70 +11,55 @@
 import android.widget.Toast;
 
 import com.example.paritosh.stormy.databinding.ActivityMainBinding;
-import com.example.paritosh.stormy.model.CurrentWeather;
 import com.example.paritosh.stormy.model.CurrentWeatherDataBindingModel;
 
-public class MainActivity extends AppCompatActivity {
+public class WeatherActivity extends AppCompatActivity {
```

Comment:
**Stage 2**
Create a _contract_ between the view and the presenter. Google _contract_ in context of MVP and you'll know what it is.